cpp/data/system_updates/2.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ -e /etc/debian_version ]; then
+    apt-get --assume-yes install fetchmail
+else
+    dnf --assumeyes install fetchmail
+fi
```

Comment:
should install_email_processor.sh be called here, as well as in installer.cc?

Reply:
No, as we don't want to install that on most of our systems.